concepticondata/conceptlists/Luangthongkum-2019-341.tsv

```diff
@@ -0,0 +1,342 @@
+Luangthongkum-2019-341-271	271	spleen	1744	SPLEEN
+Luangthongkum-2019-341-272	272	split (bamboo strip)		
+Luangthongkum-2019-341-273	273	spray (v.)		
+Luangthongkum-2019-341-274	274	spread out (a mat)	38	SPREAD OUT
+Luangthongkum-2019-341-275	275	squeeze, massage		
+Luangthongkum-2019-341-276	276	squirrel	1628	SQUIRREL
+Luangthongkum-2019-341-277	277	stand (v.)	1442	STAND
+Luangthongkum-2019-341-278	278	star	1430	STAR
+Luangthongkum-2019-341-279	279	steal (v.)	713	STEAL
+Luangthongkum-2019-341-280	280	stingy	1774	STINGY
+Luangthongkum-2019-341-281	281	stomach	819	STOMACH
+Luangthongkum-2019-341-282	282	stone, rock	2125	STONE OR ROCK
+Luangthongkum-2019-341-283	283	string, rope	1218	ROPE
+Luangthongkum-2019-341-284	284	suck	1421	SUCK
+Luangthongkum-2019-341-285	285	sun	1343	SUN
+Luangthongkum-2019-341-286	286	sunlight, hot		
+Luangthongkum-2019-341-287	287	sway, swing	573	SWAY
+Luangthongkum-2019-341-288	288	swollen, swelling		
+Luangthongkum-2019-341-289	289	tail	1220	TAIL
+Luangthongkum-2019-341-290	290	tasteless, faded	1771	TASTELESS
+Luangthongkum-2019-341-291	291	ten	1515	TEN
+Luangthongkum-2019-341-292	292	tendon	1137	TENDON
+Luangthongkum-2019-341-293	293	termite	883	TERMITE
+Luangthongkum-2019-341-294	294	termite (winged~)		
+Luangthongkum-2019-341-295	295	termite mound		
+Luangthongkum-2019-341-296	296	thick	1244	THICK
+Luangthongkum-2019-341-297	297	thin (as of cloth)	2306	THIN (OF LEAF AND CLOTH)
+Luangthongkum-2019-341-298	298	thousand	1843	THOUSAND
+Luangthongkum-2019-341-299	299	thread (a needle)		
+Luangthongkum-2019-341-300	300	three	492	THREE
+Luangthongkum-2019-341-301	301	tick	1527	TICK
+Luangthongkum-2019-341-302	302	tie (a string)	1917	TIE
+Luangthongkum-2019-341-303	303	tiger	846	TIGER
+Luangthongkum-2019-341-304	304	tilted		
+Luangthongkum-2019-341-305	305	toad	894	TOAD
+Luangthongkum-2019-341-306	306	tomorrow	1329	TOMORROW
+Luangthongkum-2019-341-307	307	tongue	1205	TONGUE
+Luangthongkum-2019-341-308	308	torn		
+Luangthongkum-2019-341-309	309	tortoise	895	TORTOISE
+Luangthongkum-2019-341-310	310	tread	575	TREAD
+Luangthongkum-2019-341-311	311	trunk (of a tree), firewood		
+Luangthongkum-2019-341-312	312	trunk (of an elephant)	3678	ELEPHANT TRUNK
+Luangthongkum-2019-341-313	313	two	1498	TWO
+Luangthongkum-2019-341-314	314	urine	1142	URINE
+Luangthongkum-2019-341-315	315	vagina	1836	VAGINA
+Luangthongkum-2019-341-316	316	vomit (v.)	1278	VOMIT
+Luangthongkum-2019-341-317	317	vulture	1185	VULTURE
+Luangthongkum-2019-341-318	318	waist	1681	WAIST
+Luangthongkum-2019-341-319	319	wart	2665	WART
+Luangthongkum-2019-341-320	320	warm	1232	WARM
+Luangthongkum-2019-341-321	321	wash (the face)		
+Luangthongkum-2019-341-322	322	wash (clothes)	3434	WASH (CLOTHES)
+Luangthongkum-2019-341-323	323	wasp	1517	WASP
+Luangthongkum-2019-341-324	324	water	948	WATER
+Luangthongkum-2019-341-325	325	wear (a shoulder bag)	413	CARRY ON SHOULDER
+Luangthongkum-2019-341-326	326	weave (cloth), loom	133	WEAVE
```

Comment:
With this referring to both the verb and the instrument, I would vote for unmapping it.

Reply:
See comment here: https://github.com/concepticon/concepticon-data/pull/876#discussion_r448788843

Reply:
It didn't occur to me first but now I really think we shouldn't map entries with a clash between object- and action-like concepts.